python/bin/watchman-wait-aio

```diff
@@ -0,0 +1,222 @@
+#!/usr/bin/env python3
+
+from __future__ import print_function
+
+from pywatchman.aioclient import AIOClient as WatchmanClient
+
+import argparse
+import asyncio
+import pywatchman
+import os
+import sys
+
+def fieldlist(s):
+    # helper for splitting a list of fields by comma
+    return s.split(',')
+
+parser = argparse.ArgumentParser(
```

Comment:
Typically you'd not mix function definitions with module-level executed code. Rather, you'd define your functions and create a main() function, that you'd call at the end using a so-called `ifmain` block:
```
if __name__ == '__main__':
    main()
```

That way we can unit test the functions in the module by importing it from somewhere else without running the script code accidentally.